templates/agent/systemd.puppet-run.timer.erb

```diff
@@ -5,7 +5,7 @@
 Description=Systemd Timer for Puppet Agent
 
 [Timer]
-OnCalendar=*-*-* <%= Array(@_hour).join(',') %>:<%= Array(@_minute).join(',') %>:00
+OnCalendar=*-*-* <%= Array(@_hour).join(',') %>:<%= Array(@_minute).join(',') %>:00 <%= @timezone %>
```

Comment:
I'd like to minimize the changes on updates. By default timezone will be empty so I wonder what's the least ugly way to avoid the trailing whitespace when it's empty.